hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/VectorClockTimestamp.java

```diff
@@ -129,7 +129,7 @@ static VectorClockTimestamp copyVector(VectorClockTimestamp vectorClockTimestamp
         return new VectorClockTimestamp(clocks);
     }
 
-    static boolean happenedBefore(VectorClockTimestamp x, VectorClockTimestamp y) {
+    public static boolean happenedBefore(VectorClockTimestamp x, VectorClockTimestamp y) {
```

Comment:
Well, that piece of code needs a refactor, since boolean return value makes it hard to use, which I'm not going to do in this PR. But I'll bear in my mind your suggestion when I'm doing those changes.